packaging/create_rpm

```diff
@@ -49,6 +51,8 @@ INSTALLER_REPO_ARCHIVE = (
     'archive/{branch}.tar.gz'
 )
 
+RPM_PREFIX = re.compile('[^0-9]+')
```

Comment:
Can we add a comment here to explain why this regex is necessary? Is it just to avoid files that start with numbers? Why?

Reply:
This is to collect the part of the filename before any version numbers, so that we can compare it to local RPMs even if the name/version number doesn't match exactly (e.g. `cloudify-rest-service-` from `cloudify-rest-service-4.3-0.1.blabla.rpm`)

Reply:
Comment added